new-website/src/pages/partner/index.jsx

```diff
@@ -5,13 +5,13 @@ import { UserInfo } from "../../components/account/user-info"
 import { LocationOverview } from "../../components/location"
 import { Invites } from "../../components/partner/invites"
 
-export const PartnerPage = () => {
+export const PartnerPage = ({ location }) => {
     return (
         <Layout>
             <Protected loginUrl="/partner/login">
                 <UserInfo />
                 <Invites />
-                <LocationOverview />
+                <LocationOverview state={location?.state} />
```

Comment:
Please do not forward a components state reference to its children. This will cause nasty side effects

Reply:
Ok, will change that and instead pass on the `saved` property that indicates success/failure when navigating back from the edit page.